src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -45,5 +45,29 @@ public static IWebJobsBuilder AddEventHubs(this IWebJobsBuilder builder, Action<
 
             return builder;
         }
+
+        public static void ConfigureOptions(EventHubOptions options)
```

Comment:
`public private void ConfigureOptions(EventHubOptions options)` ?

Reply:
Changed to internal.  The method is just a helper and does not change members of the class plus I need to call it from the tests.